### PR TITLE
Setting Employee Mapping error, resolving and export exception decorator

### PR DESCRIPTION
**Pull request by Ashutosh619-sudo**

### CODE REVIEW

File: apps/netsuite/exceptions.py [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+netsuite_error_message = 'NetSuite System Error'
+
+def __handle_netsuite_connection_error(expense_group: ExpenseGroup, task_log: TaskLog) -> None:
+    logger.info(
+        'NetSuite Credentials not found for workspace_id %s / expense group %s',
+        expense_group.id,
+        expense_group.workspace_id
+    )
+    detail = {
+        'expense_group_id': expense_group.id,
```
Comment (ashwin1111) [inline apps/netsuite/exceptions.py:27]: this is not needed, can treat payment and exports same

File: apps/netsuite/exceptions.py [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+                        'NetSuite Credentials not found for workspace_id %s',
+                        workspace_id
+                    )
+                    detail = {
+                        'message': 'NetSuite Account not connected'
+                    }
+                    task_log.status = 'FAILED'
+                    task_log.detail = detail
+
+                    task_log.save()
```
[the remark this answers is not in the document]
Comment (Ashutosh619-sudo) [reply apps/netsuite/exceptions.py:109]: in the payment function there is no expense_group

[the remark this answers is not in the document]
Comment (ashwin1111) [reply apps/netsuite/exceptions.py:109]: https://github.com/fylein/fyle-netsuite-api/pull/445#discussion_r1405686117

[the remark this answers is not in the document]
Comment (Ashutosh619-sudo) [reply apps/netsuite/exceptions.py:109]: also the Error Model needs expense_group

[the remark this answers is not in the document]
Comment (ashwin1111) [reply apps/netsuite/exceptions.py:88]: can assign expense_group = None inside 55 line block and then in __handle_netsuite_connection_error(), check for expense_group existence and update or create things

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**